sensors/test_deps.go

```diff
@@ -254,7 +311,7 @@ func getIMUWithInvalidProperties() *inject.MovementSensor {
 	imu.PropertiesFunc = func(ctx context.Context, extra map[string]interface{}) (*movementsensor.Properties, error) {
 		return &movementsensor.Properties{
 			AngularVelocitySupported:    false,
-			LinearAccelerationSupported: false,
```

Comment:
Why false -> true for invalid properties sensor?

Reply:
I made this change because this specific test IMU sensor is used in a test that's declaring `"Failed IMU creation with sensor that does not support AngularVelocity"`. Thought changing `LinearAccelerationSupported` to `true` reflects better what the test is describing.